core/controller/src/main/scala/whisk/core/controller/Namespaces.scala

```diff
@@ -129,18 +113,8 @@ trait WhiskNamespacesApi
      * - 401 Unauthorized
      * - 500 Internal Server Error
      */
-    private def getNamespaces(user: Subject)(implicit transid: TransactionId) = {
-        onComplete(iam.namespaces(user)) {
-            case Success(namespaces) =>
-                logging.info(this, s"[GET] namespaces success: $namespaces")
-                complete(OK, namespaces)
-            case Failure(r: RejectRequest) =>
-                logging.info(this, s"[GET] namespaces failed: ${r.message}")
-                terminate(r.code, r.message)
-            case Failure(t) =>
-                logging.error(this, s"[GET] namespaces failed: ${t.getMessage}")
-                terminate(InternalServerError)
-        }
+    private def getNamespaces(user: Identity)(implicit transid: TransactionId) = {
+        complete(OK, List(user.namespace))
```

Comment:
In the foreseeable future probably not. Didn't want too much of a semantic change in here though. That way, behaviour is at least theoretically preserved, as in: You can access every namespace in that list with that key.